DataFormats/L1Trigger/interface/HOTPDigiTwinMux.h

```diff
@@ -0,0 +1,91 @@
+#ifndef HOTPDIGI_TWINMUX_H
+#define HOTPDIGI_TWINMUX_H
+
+#include <boost/cstdint.hpp>
+#include <ostream>
+#include "DataFormats/HcalDetId/interface/HcalDetId.h"
+
+/** \class HOTPDigiTwinMux
+  *  Simple container packer/unpacker for HO TriggerPrimittive in TwinMUX
+  *  Trigger Primitive from HO HTR
+  *
+  *  \author Saxena, Pooja - DESY
+  */
+
+class HOTPDigiTwinMux {
+ public:
+  typedef HcalDetId key_type; /// For the sorted collection
+
+  HOTPDigiTwinMux() {theTP_HO=0;}
+  HOTPDigiTwinMux(uint64_t data) {theTP_HO = data;}
```

Comment:
please add a short summary of how the bits of theTP_HO are packed (here or in a README.md)